tests/rgw/sanity_rgw.py

```diff
@@ -94,6 +97,22 @@ def run(ceph_cluster, **kw):
             sudo=True, cmd=f"yum install -y {pkgs}", long_running=True
         )
 
+    if install_start_kafka:
+        log.info("install and start kafka broker for bucket notification tests")
+        rgw_node.exec_command(
+            cmd="wget http://magna002.ceph.redhat.com/cephci-jenkins/kafka_2.13-2.8.0.tgz && tar -zxvf kafka_2.13-2.8.0.tgz && mv kafka_2.13-2.8.0 kafka && chown cephuser:cephuser kafka", long_running=True
+        )
+        KAFKA_HOME='/home/cephuser/kafka/'
```

Comment:
It is a good practice to a space i.e. `KAFKA_HOME = "/usr/local/kafka"`

Reply:
sure, changed the default location